Makefile.include

```diff
@@ -713,21 +713,24 @@ flash-only: $(FLASHDEPS)
 preflash: $(BUILD_BEFORE_FLASH)
 	$(PREFLASHER_PREFIX)$(PREFLASHER) $(PREFFLAGS)
 
+ifneq (,$(TERMLOG))
+  TERMTEE = | tee -a $(TERMLOG)
```

Comment:
```suggestion
  TERMTEE ?= | tee -a $(TERMLOG)
```

Reply:
Is there also a use case where you would only want to log?

Reply:
Mhhh.. I can not think of one. Even if you completely automate you most of the time want to check the output.

Reply:
Why `?=` btw?

Reply:
(you can already manipulate it  via `TERMLOG` so I'm not sure if making it overridable makes sense)

Reply:
> Why `?=` btw?

You might not want to append but to overwrite the log file (so no `-a`), at least that is what I though off

Reply:
> Mhhh.. I can not think of one. Even if you completely automate you most of the time want to check the output.

Ok lets leave it like this then.

Reply:
> > Why `?=` btw?
> 
> You might not want to append but to overwrite the log file (so no `-a`), at least that is what I though off

Ah yeah if you e.g. want to prepend a timestamp to the logged content, that would be helpful